test/conftest.py

```diff
@@ -263,54 +261,6 @@ def ferc1_xbrl_sql_engine(ferc_to_sqlite_xbrl_only, dataset_settings_config):
     return ferc1_xbrl_sqlite_io_manager(context).engine
 
 
-@pytest.fixture(scope="session")
-def ferc_xbrl(
-    live_dbs,
-    ferc_to_sqlite_settings,
-    pudl_datastore_fixture,
-):
-    """Extract XBRL filings and produce raw DB+metadata files.
-
-    Extracts a subset of filings for each form for the year 2021.
-    """
-    if not live_dbs:
-        year = 2021
-
-        # Prep datastore
-        datastore = FercXbrlDatastore(pudl_datastore_fixture)
-
-        # Set step size for subsetting
-        step_size = 5
-
-        for form in XbrlFormNumber:
-            raw_archive, taxonomy_entry_point = datastore.get_taxonomy(year, form)
-
-            sqlite_engine = _get_sqlite_engine(form.value, True)
-
-            form_settings = ferc_to_sqlite_settings.get_xbrl_dataset_settings(form)
-
-            # Extract every fifth filing
-            filings_subset = datastore.get_filings(year, form)[::step_size]
-            xbrl.extract(
-                filings_subset,
-                sqlite_engine,
-                raw_archive,
-                form.value,
-                requested_tables=form_settings.tables,
-                batch_size=len(filings_subset) // step_size + 1,
-                workers=step_size,
-                # TODO(janrous): the following should ideally be provided by some
-                # ferc dataset metadata object rather than encoding this in settings.
-                datapackage_path=PudlPaths().output_file(
-                    f"ferc{form.value}_xbrl_datapackage.json"
-                ),
-                metadata_path=PudlPaths().output_file(
-                    f"ferc{form.value}_xbrl_taxonomy_metadata.json"
-                ),
-                archive_file_path=taxonomy_entry_point,
-            )
-
-
```

Comment:
Without this fixture, how are the FERC XBRL databases being generated for use in the ETL tests, and how are we doing integration testing to ensure that we're able to extract data from all the forms? Is this just cruft that's been replaced by other fixtures now?

Reply:
These guys are being generated by the `ferc_to_sqlite_xbrl_only` fixture, now.